src/auto_splitting/mod.rs

```diff
@@ -944,6 +945,10 @@ fn watchdog<T: event::CommandSink + TimerQuery + Send>(shared_state: Arc<SharedS
                 };
 
                 if result.timed_out() {
+                    if !has_timed_out {
+                        log::error!(target: "Auto Splitter", "timeout, no update in {} seconds", TIMEOUT.as_secs_f32());
+                        has_timed_out = true;
```

Comment:
This never gets reset to false again. The rest looks good.

Reply:
Btw, long term I think we want the watchdog to be more configurable (the timeout) and with a custom callback where the UI could then for example ask the user if they want to shut down the auto splitter.

Reply:
Thanks for the catch, fixed in `timeout: reset to false again`.